apps/src/blockTooltips/DropletAutocompleteParameterTooltipManager.js

```diff
@@ -88,15 +88,21 @@ DropletAutocompleteParameterTooltipManager.prototype.onCursorMovement_ = functio
 DropletAutocompleteParameterTooltipManager.prototype.showParamDropdownIfNeeded_ = function (editor, paramInfo) {
   // Check the dropletConfig to see if we can find dropdown info for this parameter
   var dropdownList;
+  // TODO: Use getAllAvailableDropletBlocks() if we need dropdowns on
+  // builtinConfigBlocks or standardConfig blocks
   this.dropletTooltipManager.dropletConfig.blocks.forEach(function (block) {
-    if (block.func === paramInfo.funcName && block.dropdown) {
+    if (!block.noAutocomplete &&
+        (!this.dropletTooltipManager.blockFilter ||
+          typeof this.dropletTooltipManager.blockFilter[block.func] !== 'undefined') &&
+        block.func === paramInfo.funcName &&
+        block.dropdown) {
```

Comment:
I did not consider the case where someone made a variable named defined. This seems like a silly enough thing to do that we shouldn't need to defend against it. My personal preference is that `this.foo !== undefined` is easier to read/understand than `typeof this.foo !== 'undefined'`, but I guess both approaches are ok.